bin/helpers/docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Uploads already created Docker image to Docker Hub
+#
+# Usage:
+# > docker_release_image <version> [tag..]
+#
+# Uploads specific version:
+# > docker_release_image mysteriumnetwork/mysterium-node alpine 0.0.1
+#
+# Uploads several versions
+# > docker_release_image mysteriumnetwork/mysterium-node ubuntu ${VERSION}-ubuntu ubuntu
+docker_release_image () {
+    DOCKER_IMAGE=$1
```

Comment:
use this approach for consuming bash parameters:
DOCKER_IMAGE=$1; shift;
DOCKER_BUILD_TAG=$1; shift;

this way you will keep param consuming and shifting in sync

Reply:
Good one. Fixed